--- a/tests/test_remap_pqm.F90	2026-09-29 19:24:36.400051859 -0500
+++ b/tests/test_remap_pqm.F90	2026-09-29 19:24:36.431051386 -0500
@@ -341,7 +341,6 @@
       end do
 
       !$omp target data map(to: dz_old, dz_new, q_in) map(from: q_out)
-      !$omp target teams distribute parallel do
       do concurrent(i=1:ncol)
          call remap_column_pqm(nz, dz_old, dz_new, q_in(:, i), q_out(:, i))
       end do
